action.yml

```diff
@@ -17,78 +17,75 @@ inputs:
     description: "API account token to retrieve your remote Codacy configuration for the project being analyzed"
   codacy-api-base-url:
     required: false
-    description: "Codacy's API URL, to retrieve your remote Codacy configuration"
+    description: "Codacy API URL to retrieve your remote Codacy configuration"
   format:
     required: false
-    description: "Format of the output file"
+    description: "Output file format"
   output:
     required: false
-    description: "Send analysis results to an output file"
+    description: "Path to a file to save the analysis results"
   directory:
     required: false
-    description: "The directory to be analyzed"
+    description: "Directory to analyze"
   parallel:
     required: false
     description: "Number of tools to run in parallel"
   max-tool-memory:
     required: false
-    description: "Maximum of allowed memory for each tool execution"
+    description: "Maximum allowed memory for each tool (bytes)"
   max-allowed-issues:
     required: false
     default: "2147483647"
     description: "Maximum number of issues allowed for the analysis to succeed"
   tool:
     required: false
-    description: "The name of the tool to analyze the code"
+    description: >-
+      Only run a specific tool or tool category (metrics, issues, duplication).
+      For a full list, see https://docs.codacy.com/repositories-configure/codacy-configuration-file/#which-tools-can-be-configured-and-which-name-should-i-use
   tool-timeout:
     required: false
-    description: "Maximum time each tool has to execute"
+    description: "Tool execution timeout (e.g. 15minutes, 1hour)"
   upload:
     required: false
     description: "Upload analysis results to Codacy"
   upload-batch-size:
     required: false
-    description: "Maximum number of results in each batch to upload to Codacy"
+    description: "Maximum number of uploaded results per batch"
```

Comment:
I'm afraid that like this, we omit the information that the "batches of results" are **uploaded to Codacy**. It's a small detail, but it does help to clarify that this has to do with the communication between the GitHub Action and Codacy (this option is uses to overcome timeout issues).

Reply:
OK, reverting